_data/issues/combo-box.yml

```diff
@@ -1,8 +1,11 @@
 title: Combo box
 type: component
+severity: moderate
 items:
-- affectsScreenReaders: true
-  summary: Known assistive technology issues.
-  summaryAdditional: Testing with people using assistive technology revealed usability concerns that require additional investigation. At this time, consider using a <a href="{{ site.baseurl }}/components/select">Select component</a> instead of a combo box. More research and testing is planned to better understand and address these accessibility issues.
-  githubRepo: uswds
-  githubNumber: 5522
+  - affectsScreenReaders: true
+    summary: Known assistive technology issues.
+    summaryAdditional: |-
+      Testing with people using assistive technology revealed usability concerns that require additional investigation.
+    recommendation: Consider using a <a href="/components/select">Select component</a> instead of a combo box. More research and testing is planned to better understand and address these accessibility issues.
```

Comment:
## Todo

Will need to find a method to link to other parts of the site. Using `site.baseurl` returns CircleCI error:

<img width="800" alt="image" src="https://github.com/uswds/uswds-site/assets/3385219/0b40f5bf-406b-4eb7-9e1e-3a8e92ce49b2">